dali/python/CMakeLists.txt

```diff
@@ -32,7 +32,8 @@ set_target_properties(${dali_python_lib} PROPERTIES
 
 # Add install rule
 install(DIRECTORY "${PROJECT_SOURCE_DIR}/dali/python/nvidia" DESTINATION "${PROJECT_BINARY_DIR}/dali/python")
-install(FILES "${PROJECT_SOURCE_DIR}/dali/python/setup.py" DESTINATION "${PROJECT_BINARY_DIR}/dali/python")
+configure_file("${PROJECT_SOURCE_DIR}/dali/python/setup.py.in" "${PROJECT_BINARY_DIR}/dali/python/setup.py")
+install(FILES "${PROJECT_BINARY_DIR}/dali/python/setup.py" DESTINATION "${PROJECT_BINARY_DIR}/dali/python")
 install(FILES "${PROJECT_SOURCE_DIR}/tools/rec2idx.py" DESTINATION "${PROJECT_BINARY_DIR}/dali/python")
 install(FILES "${PROJECT_SOURCE_DIR}/tools/tfrecord2idx" DESTINATION "${PROJECT_BINARY_DIR}/dali/python")
```

Comment:
Not for this PR, but we really need to fix this file so that the necessary files to `pip install` from are copied into `${PROJECT_BINARY_DIR}/dali/python`  at build time rather than install time.  As it is, I'm pretty sure we basically force people to set the install path the build directory.

Reply:
I will figure out something.

Reply:
How about https://github.com/NVIDIA/DALI/pull/87 ?